client/pages/sources/show/access/access.js

```diff
@@ -31,18 +35,40 @@ const helpers = {
     }
     return '';
   },
+  checkedIfAccessToSourceCanBeRequested() {
+    const source = Sources.findOne({ _id: FlowRouter.getParam('_id') });
+
+    return source.isRequestable ? 'checked' : '';
```

Comment:
Makes sense. I like [`_.get`](https://lodash.com/docs/4.17.4#get) for this (particularly when the property path is long). Ok with using it?